android/src/main/java/com/pspdfkit/react/ReactPdfViewManager.java

```diff
@@ -112,9 +112,36 @@ public void setFragmentTag(PdfView view, @NonNull String fragmentTag) {
     }
 
     @ReactProp(name = "configuration")
-    public void setConfiguration(PdfView view, @NonNull ReadableMap configuration) {
+    public void setConfiguration(@NonNull PdfView view, @NonNull ReadableMap configuration) {
         ConfigurationAdapter configurationAdapter = new ConfigurationAdapter(view.getContext(), configuration);
-        view.setConfiguration(configurationAdapter.build());
+        view.setConfiguration(updateReactPropsInConfiguration(configurationAdapter.build()));
+    }
+
+    @ReactProp(name = "enableFormEditing")
+    public void setEnableFormEditing(@NonNull PdfView view, boolean enableFormEditing) {
+        this.isFormEditingEnabled = enableFormEditing;
+
+        final PdfActivityConfiguration configuration = view.getConfiguration();
+        final PdfActivityConfiguration updatedConfiguration = updateReactPropsInConfiguration(configuration);
+        if (!updatedConfiguration.equals(configuration)) {
+            view.setConfiguration(updatedConfiguration);
+        }
+    }
+
+    @NonNull
+    private PdfActivityConfiguration updateReactPropsInConfiguration(@NonNull PdfActivityConfiguration configuration) {
```

Comment:
The reasoning behind this method is that iOS defines the `enableFormEditing` property as well as the configuration. Since the order of react props application is not guaranteed, we need to make sure that the property value is not overriden by the configuration.

Reply:
On iOS, the last used option overrides the previous one. For example, if the user sets a value in the configuration, and they later override it in the prop, we will use the most recent one.